src/trace-plugin-interface.js

```diff
@@ -102,7 +102,6 @@ RootSpan.prototype.getTraceContext = function() {
 /**
  * PluginAPI constructor. Don't call directly - a plugin object will be passed to
  * plugin themselves
- * TODO(kjin): Should be called something else
```

Comment:
Though it's been renamed since I added this TODO (from `Plugin` to `PluginAPI`), I still think this should be renamed, maybe to `TraceInterface` or something?